templates/boltzmann/middleware/honeycomb.ts

```diff
@@ -108,14 +183,33 @@ function honeycombMiddlewareSpans ({name}: {name?: string} = {}) {
     return (next: Handler) => (context: Context) => next(context)
   }
 
-  return function honeycombSpan (next: Handler) {
+  if (process.env.HONEYCOMB_API_HOST) {
```

Comment:
this can't be used to pivot between the two types right? We have that var defined in existing servers

Reply:
I started a refactor to wrap this logic in an `isOtel` function but it looks like I missed a spot. Good eye!

Reply:
This should 🤞 be fixed throughout.